var/spack/repos/builtin/packages/morphio/package.py

```diff
@@ -25,10 +25,15 @@ class Morphio(CMakePackage):
     depends_on('cmake@3.2:', type='build')
     depends_on('hdf5~mpi', when='~mpi')
     depends_on('hdf5+mpi', when='+mpi')
```

Comment:
Do we need hdf5 here then? I assume morphio doesn’t use h5 directly but via highfive only

Reply:
I guess technically not. It shipped it's own version of HighFive before…